addons/config/expected/cpi.tanzu.vmware.com/v1alpha1/vspherecpiconfig/tkgs.yaml

```diff
@@ -6,3 +6,21 @@ metadata:
 spec:
   vsphereCPI:
     mode: vsphereParavirtualCPI
+---
+apiVersion: cpi.tanzu.vmware.com/v1alpha1
+kind: VSphereCPIConfig
+metadata:
+  name: v1.23.3---vmware.1-tkg.1-routable
+  namespace: tkg-system
+spec:
+  vsphereCPI:
+    mode: vsphereParavirtualCPI
+    template:
```

Comment:
where's this template field coming from? i didn't see the definition https://github.com/vmware-tanzu/tanzu-framework/blob/50c237ec19e948f4999679ec44d5eb6b8ca407dd/apis/cpi/v1alpha1/vspherecpiconfig_types.go#L102